tools/tpm2_nvwrite.c

```diff
@@ -233,9 +239,10 @@ bool tpm2_tool_onstart(tpm2_options **opts) {
         { "offset",         required_argument, NULL, 'o' },
         { "set-list",       required_argument, NULL, 'L' },
         { "pcr-input-file", required_argument, NULL, 'F' },
+        { "hmac-auth",      no_argument, NULL, 'X' },
```

Comment:
You should also add this new option to the man page.

Reply:
And also you should add a unit test for HMAC auth.

Reply:
@martinezjavier the man page for authorization is now updated. Not really sure if i can add a unit test that will work. Reason being, the session continuation is broken with user space resource manager and kernel resource manager is not available with simulator. Currently am testing it working directly with the TPM.